openfisca_uk/variables/finance/benefit/family/benefit_cap.py

```diff
@@ -50,9 +53,9 @@ def formula(benunit, period, parameters):
             "ESA_contrib",
         ]
         qualifying = (
-            add(benunit, period, ["working_tax_credit"], options=[DIVIDE])
-            + add(benunit, period, ["ESA_income"], options=[ADD])
-            + aggr(benunit, period, QUAL_PERSONAL_BENEFITS, options=[ADD])
+            add(benunit, period, ["working_tax_credit"])
```

Comment:
To recap here, it's because by making everything yearly, we've shifted from doing the adding/dividing time periods in this specific variables to doing the same operations within `working_tax_credit`, `ESA_income` and the others.